topics/introduction/tutorials/galaxy-intro-101/tutorial.md

```diff
@@ -602,7 +598,7 @@ Now that we have built our workflow, let's use it on some different data. For ex
 >    >  * If you see an "Invalid column choice" error, you need to specify which column you want to use. If you have to type the column number, you need to type just the number e.g. `4` (not `Column 4` or anything else).
 >    {: .comment}
 >
-> 8. Once the workflow has started, you will initially be able to see all its steps, but the unimportant intermediates will disappear after they complete successfully:
+> 7. Once the workflow has started, you will initially be able to see all its steps, but the unimportant intermediates will disappear after they complete successfully:
```

Comment:
fyi the numbering is automatic so no real need to fix these unless they bother you